spring-kafka/src/main/java/org/springframework/kafka/core/StreamsBuilderFactoryBean.java

```diff
@@ -140,12 +144,21 @@ public void setClientSupplier(KafkaClientSupplier clientSupplier) {
 		this.clientSupplier = clientSupplier; // NOSONAR (sync)
 	}
 
+	public void addKafkaStreamsCustomizers(List<KafkaStreamsCustomizer> kafkaStreamsCustomizers) {
```

Comment:
This must be setter. Remember XML `<bean>`. 
And as I said before: only single `KafkaStreamsCustomizer` should be injected here. The list-like behavior we can reach with the `CompositeKafkaStreamsCustomizer` implementation.